docker/container.go

```diff
@@ -174,6 +191,16 @@ func (cd *ContainerDetails) GetResources() util.Resources {
 	return cd.Resources
 }
 
+// GetTestNetID gets the testnet ID for the given container
+func (cd *ContainerDetails) GetTestNetID() string {
```

Comment:
These two functions should just be a GetLabels() function, which returns a map[string]string